extensions/2.0/Vendor/EXT_feature_metadata/schema/featureIdTexture.schema.json

```diff
@@ -2,21 +2,24 @@
   "$schema": "http://json-schema.org/draft-04/schema",
   "title": "Feature ID Texture",
   "type": "object",
+  "allOf": [ { "$ref": "textureInfo.schema.json" } ],
   "description": "An object describing a texture used for storing per-texel feature IDs.",
   "properties": {
+    "index": { },
+    "texCoord": { },
     "featureTable": {
       "type": "string",
       "description": "The ID of the feature table in the model's root `EXT_feature_metadata.featureTables` dictionary."
     },
-    "featureIds": {
-      "allOf": [ { "$ref": "textureAccessor.schema.json" } ],
-      "description": "A description of the texture and channel to use for feature IDs. The `channels` property must have a single channel. Furthermore, feature IDs must be whole numbers in the range `[0, count - 1]` (inclusive), where `count` is the total number of features in the feature table. Texel values must be read as integers. Texture filtering should be disabled when fetching feature IDs."
+    "channel": {
+      "type": "integer",
+      "minimum": 0,
+      "description": "Single channel index storing per-texel feature IDs."
     },
     "extensions": {},
     "extras": {}
   },
   "required": [
-    "featureTable",
-    "featureIds"
+    "featureTable"
```

Comment:
Should `channel` be listed as required?

Reply:
Good point – I believe this property should be required, yes. ✅